InAppUtils/InAppUtils.m

```diff
@@ -118,13 +141,13 @@ - (void)paymentQueue:(SKPaymentQueue *)queue
         switch (error.code)
         {
             case SKErrorPaymentCancelled:
-                callback(@[@"user_cancelled"]);
+                callback(@[RCTMakeError(@"user_cancelled", nil, nil)]);
                 break;
             default:
-                callback(@[@"restore_failed"]);
+                callback(@[RCTJSErrorFromNSError(error)]);
```

Comment:
this is going to be a breaking change. I think we can avoid making a breaking change with this PR.

Reply:
I'm still leaving this for now in case we do make other breaking changes, then we might as well add this too.